### PR TITLE
Always fill UDP checksums in DHCP replies in VR

### PR DESCRIPTION
**Pull request by remibergsma**
Backport of PR 1743 of ACS

### CODE REVIEW

File: cosmic-core/systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py [hunk 1]
```diff
@@ -376,6 +377,8 @@ def fw_router(self):
             self.fw.append(
                 ["filter", "", "-A INPUT -i %s -p udp -m udp --dport 67 -j ACCEPT" % self.dev])
             self.fw.append(
+                ["mangle", "front", "-A POSTROUTING " + "-p udp -m udp --dport 68 -j CHECKSUM --checksum-fill"])
```
Comment (borisroman) [inline cosmic-core/systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py]: This could be removed! @remibergsma

**Comment by remibergsma** [comment]
go build

**Comment by remibergsma** [comment]
go build

**Comment by borisroman** [comment]
go build

**Pull request merged**